benchmarks/core/reverse.bril

```diff
@@ -0,0 +1,31 @@
+
+@main{
+  input: int = const 123;
```

Comment:
This is OK, but to make the benchmark a little more reusable, you might consider making the input a parameter. Here's an example:
https://github.com/sampsyo/bril/blob/a7a0759b20e724260825fc9c6ae7273a09c0bafd/benchmarks/core/armstrong.bril#L1-L2

That is, you can make `input` a parameter to `@main` and then give it a value like `# ARGS: 123` for benchmark automation purposes.